windows/deploy_windows.ps1

```diff
@@ -275,7 +271,32 @@ Function Build-Installer
         "$WindowsPath\installer.nsi")
 }
 
+# Build and copy NS-Process dll
+Function Build-NSProcess
```

Comment:
Ok. I now see that you've implemented my PR ;-). Thanks!

Reply:
so now I see you've found the implementation already :)